src/main/kotlin/com/milaboratory/mixcr/basictypes/GeneFeatures.kt

```diff
@@ -24,7 +24,7 @@ import io.repseq.core.GeneFeature
 @Serializable(by = GeneFeatures.SerializerImpl::class)
 data class GeneFeatures(
     @JsonValue val features: List<GeneFeature>
-) {
+) : Iterable<GeneFeature> by features {
```

Comment:
I didn't do it because it overlaps with intersection and plus functions of this class